packages/website/bin/handle-config.js

```diff
@@ -26,6 +26,7 @@ const defaultConfig = {
   packages: ['./packages/*'],
   useManifests: false,
   webpack: identityFunc,
+  showExamples: true,
```

Comment:
Can we default this to false? See also, the other place we default this.

I think this should be a fairly uncommon pattern, but we need it to support Jira.